changelogs/unreleased/4130-v1tov2-does-not-specify-packages-sufficiently-strict.yml

```
@@ -0,0 +1,6 @@
+description: Limit included namespace packages to inmanta_plugins
+issue-nr: 4130
+change-type: patch
+destination-branches: [iso5, master]
+sections: {
```

```suggestion
sections: {
  bugfix: "{{description}}"
```

This is a potentially user visible issue so I would include a bugfix note.